app/crate/genre_descriptions.py

```
@@ -0,0 +1,678 @@
+
+    urls_to_try = [
+        f"https://musicbrainz.org/genre/{mbid}",
+        f"https://musicbrainz.org/genre/{mbid}/details",
+    ]
+    wikidata_url = ""
+    for url in urls_to_try:
+        try:
+            _throttle_musicbrainz()
+            page_html = _request_text(url, headers=_mb_headers())
+        except Exception:
+            continue
```

Ruff (0.15.9)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)